app/middlewares/validateSchema.js

```diff
@@ -0,0 +1,20 @@
+const { validationResult, checkSchema } = require('express-validator');
+
+const logger = require('../logger');
+const { validationError } = require('../errors');
+
+const validateSchema = schema => [
+  checkSchema(schema),
+  (req, _, next) => {
+    const errors = validationResult(req);
+    if (errors.isEmpty()) next();
+    else {
+      const errorMessage = errors.array()[0].msg;
+
+      logger.error(`Validation error: ${req.path}: ${errorMessage}`);
+      next(validationError(errorMessage));
```

Comment:
Here you are just throwing the first error. Could you response all the validation errors or there is some limitation and/or requirement to not do so?

Reply:
I did that to make it more easy to catch. In muy personal opinion, in most cases is easier to the front-end to receive one error. But i change it in order to receive all errors.